website/content/partials/components/platform-nomad-jobspec-canary.mdx

```diff
@@ -1,142 +1,11 @@
 ## nomad-jobspec-canary (platform)
 
-Deploy to a Nomad cluster from a pre-existing Nomad job specification file.
```

Comment:
Do we need to add a new partial mdx file to account for the plugin moving? I think the docs are still around, just not being included because it's a new partial?

Reply:
Sweet!